### DISCUSSION TITLE
请问windows的5.44能出个补丁吗？

### DISCUSSION THREAD

**Discussion by tsgs912** [Cirn09/calibre-do-not-translate-my-path#39] [Q&A]
请问windows的5.44能出个补丁吗？
我的win8系统目前只能更新到calibre的5.44，6x7x的calibre不再支持win8。能否期待出一个5.44的补丁让win8的calibre也能跟着用一下，十分期待，万分感谢!!

**Comment by Cirn09** [comment]
https://github.com/Cirn09/calibre-do-not-translate-my-path/releases/tag/v5.44.0

**Reply by tsgs912** [reply]
好的用上了，谢谢!!
